seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/serialize/ElasticsearchRowSerializer.java

```diff
@@ -81,7 +84,7 @@ public String serializeRow(SeaTunnelRow row) {
 
     private String serializeUpsert(SeaTunnelRow row) {
         String key = keyExtractor.apply(row);
-        Map<String, Object> document = toDocumentMap(row);
+        String document = new String(serializationSchema.serialize(row));
```

Comment:
I'm confused about the usage, why use new String? I see that `String serialize(SeaTunnelRow row);` return String

Reply:
<img width="889" alt="image" src="https://github.com/apache/seatunnel/assets/51053924/f4cb5a9f-b262-4134-a2a4-72f5be21874f">